websubmit/bibtex_unit_test.py

```diff
@@ -189,6 +189,19 @@ def test_search_texkey(self):
         self.assertTrue(ret.index(r'@article'))
 
 
+    def test_search_adsbibcode(self):
```

Comment:
so this is a positive test. it ensures that the changes match one correct ADS bibcode
how about adding a negative test? something that may looks like a bibcode, but may only have 18 characters or contain an unexpected character. tests should ensure that the regexp match is specific and narrow